tests/test_evaluate_model.py

```diff
@@ -86,7 +107,12 @@ def test_evaluate_model_multiple_thresholds(
     data_path, label_binarizer_path, model_path
 ):
     evaluate_model(
-        "mesh-cnn", model_path, data_path, label_binarizer_path, [0, 1, 0.5, 0.9]
+        "mesh-xlinear",
+        model_path,
+        data_path,
+        label_binarizer_path,
+        [0, 1, 0.5, 0.9],
+        sparse_y=False,
     )
```

Comment:
As above, are there any outputs that need to be checked? I'm just asking because test_evaluate_model has some assert statements in it and this one doesn't.

Reply:
I added in an assert